Formula/g/grok.rb

```diff
@@ -1,15 +1,23 @@
 class Grok < Formula
   desc "DRY and RAD for regular expressions and then some"
   homepage "https://github.com/jordansissel/grok"
-  url "https://github.com/jordansissel/grok/archive/refs/tags/v0.9.2.tar.gz"
-  sha256 "40edbdba488ff9145832c7adb04b27630ca2617384fbef2af014d0e5a76ef636"
   license "BSD-2-Clause"
   revision 2
   head "https://github.com/jordansissel/grok.git", branch: "master"
 
+  stable do
+    url "https://github.com/jordansissel/grok/archive/refs/tags/v0.9.2.tar.gz"
+    sha256 "40edbdba488ff9145832c7adb04b27630ca2617384fbef2af014d0e5a76ef636"
+
+    # Backport Makefile fix to support compile on Ubuntu 12.04+
+    patch do
+      url "https://github.com/jordansissel/grok/commit/f440e9b4ce29a8e803f09d39e37a5725724aba95.patch?full_index=1"
+      sha256 "2f92f3b5956224c6d5674940d1d604c1aafa1c3387bda68a266e853c90d0fa86"
+    end
+  end
+
   livecheck do
-    url :stable
-    regex(/^v?(\d+\.\d{,3}(\.\d+)*)$/i)
+    skip "No longer developed or maintained"
```

Comment:
This has 230 installs in the last year with 0 formulae depending on it. Why don't we just deprecate it?

Reply:
I don't mind either way, but the exact deprecation decision for this situation is vague given our unlisted policy of keeping "mature" software.

Only Debian and MacPorts also distribute this; however, Debian Trixie has dropped it due to their effort to kill `pcre`. We may end up doing something similar, but maybe after enough upstreams are forced by Debian to update to `pcre2`.